server/tools/attestation-service-ctl/readme.md

```diff
@@ -0,0 +1,34 @@
+# attestation-service-ctl
+
+attestation-service-ctl is a command line interface for the attestation server interface. It provides the following functions:
+- Set/Get each TEE's Open Policy Engine(OPA) `Policy(.rego)` and `Reference Data(.json)` files. The `Reference Data(.json)` contents should come from such as Reference Value Provider Service(RVPS), but this item is not reflected in this implementation.
+- Restore each TEE's `Policy(.rego)` and `Reference Data(.json)` to default value.
+- Provide the Attestation Server's `attestation` endpoint testing functionality.
+
+## Supported TEEs
+
+Currently the `attestation-service-ctl` supports the following types of TEE:
+- sgx
+- tdx
+- sevsnp
+- sample: The TEE used to demo/test Attestation Server's functionalities.
```

Comment:
What does this actually mean though? Is it a "dummy" / nop TEE? Maybe the name should make it's functionality (or lack of?) clearer to avoid confusion.